spec/models/book_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.describe Book, type: :model do
+  describe 'validations' do
+    it { should validate_presence_of :title }
+    it { should validate_presence_of :description }
+    it { should validate_presence_of :image }
+    it { should validate_presence_of :est_reading_time }
+    it { should validate_presence_of :page_count} 
+  end
```

Comment:
I know we mention this earlier, but just making a note that we should update the page_count to validate > 0 here :)